src/scripting/Convo.js

```diff
@@ -274,7 +274,11 @@ class Convo {
 
           return this.scriptingEvents.setUserInput({ convo: this, convoStep, container, scriptingMemory, meMsg: convoStep })
             .then(() => debug(`${this.header.name}/${convoStep.stepTag}: user says ${JSON.stringify(convoStep, null, 2)}`))
-            .then(() => this.scriptingEvents.onMeStart({ convo: this, convoStep, container, scriptingMemory }))
+            .then(() => {
+              const meMsg = new BotiumMockMessage(convoStep)
```

Comment:
I think this should actually happen before line 275, and meMsg should then always be handed over from transcriptStep.actual

Reply:
I tried it, but so media is not extracted from convostep. (for example test/scripting/userinputs/mediaInputConvos.spec.js fails)

Reply:
hm then maybe the unit test is defect as well. i will take care of it